Alignment/ReferenceTrajectories/src/GblTrajectory.cc

```diff
@@ -603,6 +603,7 @@ namespace gbl {
 
     int nOffset = aPoint.getOffset();
 
+    for (unsigned int i = 0; i < 5; ++i) anIndex[i] = 0;
```

Comment:
@ckleinw, @mschrode I propose to replace this with the simpler statement
```
anIndex = {};
```
This sets every non-specified element to zero, i.e. all of them.